src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -130,7 +136,7 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
       stateObj.state === "playing"
```

Comment:
I can't comment on the line above for some reason but should it be 
```js
this._showProgressBar !== false
```

Reply:
showProgressBar returns a boolean.